lua/ulx/modules/sh/uptime.lua

```diff
@@ -0,0 +1,36 @@
+local function plural(value, noun)
+	return value ~= 1 and noun .. "s" or noun
+end
+
+local startTime = SysTime()
+local uptimeCommand = ulx.command("Chat", "ulx uptime", function()
+	local time_data = string.FormattedTime(SysTime() - startTime)
+	local h, m, s = time_data.h, time_data.m, time_data.s
```

Comment:
nit: Couple of nitpicks on variable naming
- `camelCase` is generally* what we use for variable names across all of our codebases
- `h`, `m`, `s` aren't super self-documenting now that we're missing the context from the table's name

My choice here would be to rename the table to `uptime` and then do `uptime.s` etc. below to give the most context, but it's very minor and entirely my opinion